pkg/controller/operators/openshift/helpers_test.go

```diff
@@ -501,7 +438,9 @@ func TestIncompatibleOperators(t *testing.T) {
 		},
 	} {
 		t.Run(tt.description, func(t *testing.T) {
-			objs := []client.Object{tt.cv.DeepCopy()}
+			objs := []client.Object{}
```

Comment:
Any idea on why we're removing the deep copy call here?

Reply:
Because we're not referencing the CV anymore and it doesn't exists in the test case (a.k.a `tt`)